### PR TITLE
Formats and prints path latitudes and longitudes to logs

### PR DESCRIPTION
**Pull request by jamenkaye**
Please let me know if my code is messy, over complicated, or too difficult to follow, or could use any other further work. The code here could be replaced with
`rospy.loginfo("path is {}".format(localPath.getLatlons()))`,
but this causes each line in the resulting log to contain the longitude of the previous point and the latitude of the current point.

### CODE REVIEW

**Comment by tylerlum** [comment]
Great work Jamen and great review Patrick. Couple of other notes:

image
This shows your code builds and passes out standard tests (doesn't break things) :D. It also says there's a lint error, which is just a style issue. You can click details, login with github, then scroll down

image

This shows the issue. 

```
/catkin_ws/src/local-pathfinding/python/main_loop.py:220:9: E303 too many blank lines (2)
/catkin_ws/src/local-pathfinding/python/main_loop.py:226:21: E117 over-indented
/catkin_ws/src/local-pathfinding/python/main_loop.py:238:9: E265 block comment should start with '# '
```
A few style things to improve on lines 220, 226 and 238 of the main loop :)

**Comment by tylerlum** [comment]
image

I see Patrick's point, but I like this too. More of a style/preference thing. 

The code is pretty clear and easy to understand. The only part I think you could work on is the j = 14, which isn't very clear. See if it's possible to do something like.

```
        for latlon in localPath.getLatlons():
            latLonsToLog += "Lat: {}, lon {}".format(latlon.lat, latlon.lon)
```

File: python/main_loop.py [hunk 1]
```diff
@@ -216,6 +217,22 @@ def createNewLocalPath(sailbot, maxAllowableRuntimeSeconds, desiredHeadingPublis
         # Publish local path
         localPathPublisher.publish(msg.path(localPath.getLatlons()))
 
+        # Format and log local path
+        latLonsToLog = "Path latlons are:"
+
+        for i in range(len(localPath.getLatlons())):
+            if(i % LOGGED_LATLONS_PER_LINE == 0):
```
Comment (tylerlum) [inline python/main_loop.py:224]: Great use of all caps named constant :)

File: python/main_loop.py [hunk 2]
```diff
@@ -216,6 +217,22 @@ def createNewLocalPath(sailbot, maxAllowableRuntimeSeconds, desiredHeadingPublis
...
+                latLonsToLog += "\n"
+
+            latLon = localPath.getLatlons()[i]
+            latLonsToLog += "(lat: {}, lon: {})".format(latLon.lat, latLon.lon)
```
Comment (tylerlum) [inline python/main_loop.py:228]: Looks very clear and easy to understand
image

**Review by tylerlum — APPROVED** [review]
Approved! Great work!
Feel free to click this Squash and Merge button to merge it in
image

**Review by patrick-5546 — APPROVED** [review]

**Pull request merged**